app/models/__init__.py

```diff
@@ -36,8 +38,15 @@ class User(Base):
     email = Column(String(250), nullable=False)
 
     tokens = relationship('Token', back_populates='user')
-    teams = relationship('Team', secondary=user_team_table, back_populates='users')
+    _teams = relationship('Team', secondary=user_team_table, back_populates='users')
     schedules = relationship('Schedule', back_populates='user')
+
+    @property
+    def team(self) -> Optional[Team]:
+        """
+        This is a read-only property. Do not UNDER ANY CIRCUMSTANCES attempt to set this.
+        """
+        return self._teams[0] if self._teams else None
```

Comment:
What about the weird case teams has 2 elements... for... weird reasons?

Reply:
Or just yolo it? 🤔

Reply:
Unique should guarantee it won't... So EEEEH